templates/shared_services/sonatype-nexus/porter.yaml

```diff
@@ -0,0 +1,65 @@
+name: tre-shared-service-nexus
+version: 0.0.1
+description: "A Sonatype Nexus shared service"
+registry: azuretre
+credentials:
+  - name: azure_tenant_id
+    env: ARM_TENANT_ID
+  - name: azure_subscription_id
+    env: ARM_SUBSCRIPTION_ID
+  - name: azure_client_id
+    env: ARM_CLIENT_ID
+  - name: azure_client_secret
+    env: ARM_CLIENT_SECRET
+parameters:
+  - name: tre_id
+    type: string
+    description: "The ID of the parent TRE instance e.g., mytre-dev-3142"
+  - name: azure_location
+    type: string
+    description: "Azure location (region) to deploy to"
+  - name: tfstate_resource_group_name
+    type: string
+    description: "Resource group containing the Terraform state storage account"
+  - name: tfstate_storage_account_name
+    type: string
+    description: "The name of the Terraform state storage account"
+  - name: tfstate_container_name
+    type: string
+    default: "tfstate"
+    description: "The name of the Terraform state storage container"
+mixins:
+  - exec
+  - az
+  - terraform:
+      clientVersion: 1.0.4
+install:
+  - terraform:
+      description: "Deploy shared service"
+      input: false
+      vars:
+        tre_id: "{{ bundle.parameters.tre_id }}"
+        location: "{{ bundle.parameters.azure_location }}"
+      backendConfig:
+        resource_group_name: "{{ bundle.parameters.tfstate_resource_group_name }}"
+        storage_account_name: "{{ bundle.parameters.tfstate_storage_account_name }}"
+        container_name: "{{ bundle.parameters.tfstate_container_name }}"
+        key: "tre-shared-service-nexus"
```

Comment:
I think you need the tre_id in the key as well

Reply:
good catch, thanks